pkg/repositories/database.go

```diff
@@ -112,20 +111,14 @@ func createPostgresDbIfNotExists(ctx context.Context, gormConfig *gorm.Config, p
 
 	dialector := postgres.Open(getPostgresDsn(ctx, pgConfig))
 	gormDb, err := gorm.Open(dialector, gormConfig)
-	if err == nil {
```

Comment:
I changed this so it checks if it is an error first. Even though the logic is the same as before, I feel checking `if err != nil` immediately after the function creating the error is safer/more natural because this is the general Go error checking pattern.

Reply:
Sorry, ignore this - I changed it back. On second thought, the quirk in flow this introduces means it isn't really much cleaner than the original code.